key_rings.go

```diff
@@ -57,11 +59,24 @@ func (c *Client) GetKeyRings(ctx context.Context) (*KeyRings, error) {
 	return &rings, nil
 }
 
+type DeleteKeyRingQueryOption func(*http.Request)
+
+func UseForce(force bool) DeleteKeyRingQueryOption {
```

Comment:
Should the name of the function be `UseForce` or `WithForce`? The With prefix seems to be the standard or at least what is in use in other similar functions.